http_service/bugbug_http/models.py

```diff
@@ -332,3 +299,99 @@ def get_config_specific_groups(config: str) -> str:
     )
 
     return "OK"
+
+
+def schedule_tests_from_patch(base_rev: str, patch_hash: str) -> str:
+    from libmozdata import vcs_map
```

Comment:
This doesn't work anymore, we need to use Lando

Reply:
Done here 489515d042fbca4ccea5aa34ee7800e9da29a5e2, and I filed an issue to do it in libmozdata (https://github.com/mozilla/libmozdata/issues/266).